### DISCUSSION TITLE
All commits to repo with failed status UwU (simple issue with jekyll theme?)

### DISCUSSION THREAD

**Discussion by VIRTUAL-K4MI** [community/community#51591] [Repositories] [answered]
Question,

All commits to my github repo result in fail

Currently I use my github repo for
My vrchat world and my websites 

Any info on why All commits 
made from my account fail?

I think it has to do with the Jekyll theme,
It worked till recently.
![Screenshot_20230331_122352_Chrome](https://user-images.githubusercontent.com/105128490/229212241-94846245-4e43-49ae-8854-5e3a67ff6695.jpg)

![Screenshot_20230331_125020_Chrome](https://user-images.githubusercontent.com/105128490/229216746-42138186-7517-42b3-9fc3-d9371e1f4f5c.jpg)

![Screenshot_20230331_125501_Chrome](https://user-images.githubusercontent.com/105128490/229217259-20d28676-94e7-41f0-962b-57a700bd7d39.jpg)

**Comment by airtower-luna** [comment, marked as answer]
Committing worked, otherwise the commits wouldn't be in the list you see. The red X means that a job for that commit failed. Click on the X to see the jobs, and which failed. There you can also check the details for each, which should have error messages. Those should tell you what's wrong.

If you need more help with those error messages, make sure to include a link so people can easily take a look.

**Reply by VIRTUAL-K4MI** [reply]
Ah yes, I know it's something simple, 
a update in the Jekyll hacker theme or....

**Reply by VIRTUAL-K4MI** [reply]
Could you check the above line for me and tell me if something looks off?

**Reply by airtower-luna** [reply]
There's definitely a typo in your `style.scss` (missing `y`). But also the [usage instructions for the "Hacker" theme](https://github.com/pages-themes/hacker#usage) look a bit different from what you're doing.

In the future please post *links*, not screenshots. The screenshots make it unnecessarily difficult to help.

**Reply by VIRTUAL-K4MI** [reply]
Thanks, I knew it was something ridiculously simple.
